Z85.cs

```diff
@@ -108,28 +108,27 @@ public static byte[] Decode(byte[] encoded)
 		public static byte[] ToZ85Decoded(this byte[] encoded) 
 		{
 			return Decode(encoded);
-		}
-
-		public static string ToZ85Decoded(this string encoded) 
+		}
+
+        public static byte[] ToZ85Decoded(this string encoded) 
```

Comment:
Hi Richard,

Are you using tabs? I usually replace with spaces. I'll tidy up and send
another PR once I know your preferences.

I have a couple of other things I'd like to contribute, like a wrapper
for the function to generate new certificates. Would you like some unit
tests too? It might mean re-arranging the directories a bit.

Cheers, Rob

--
  Robert Pickering
  robertfpickering@fastmail.com
 
 
 
On Tue, Jul 28, 2015, at 05:42 PM, Richard Newton wrote:
> In Z85.cs[1]:


> > @@ -108,28 +108,27 @@ public static byte[] Decode(byte[] encoded)
>               public static byte[] ToZ85Decoded(this byte[] encoded) 
>               {
>                       return Decode(encoded);
> -             }
> -
> -             public static string ToZ85Decoded(this string encoded) 
> +             }
> +
> +        public static byte[] ToZ85Decoded(this string encoded)
> Tabbing seems to be a bit messed up here


> —
> Reply to this email directly or view it on GitHub[2]. 


>  
 


Links:

  1. https://github.com/zeromq/clrzmq4/pull/33#discussion_r35662562
  2. https://github.com/zeromq/clrzmq4/pull/33/files#r35662562